### ISSUE TITLE
Only applies to methods, not members

### ISSUE THREAD

**Issue by simonhaenisch** [chinchiheather/tslint-lines-between-class-members#11]
```ts
export class FooBar {
  foo;
  bar;
}
```

This doesn't report anything even though there is no new line between the members `foo` and `bar`.

If this rule is only for class **methods**, it should maybe not be named lines-between-class-**members**?

Would be nice to make it work for any type of member and then you could make it configurable in case someone only wants it to apply to methods.

**Comment by chinchiheather** [comment]
Hi @simonhaenisch, yeah I did initially have it named as `lines-between-class-methods` because it does only relate to class methods. But as I based it on the eslint rule which is called `lines-between-class-members` (and also only affects methods) I thought it would be easier for people to find if it had the same name. I was in two minds about it though.

I have no issue with your suggestion about adding this in for all class members, however I don't think I have time to get round to it right now, but if you would like to make a PR that would be awesome :) I would say it is more likely that people would not want lines between variables though, so the configuration should be to add it in rather than disable it.
